components/TripList/index.js

```diff
@@ -0,0 +1,22 @@
+import useSWR from "swr";
+import Trip from "../Trip";
+
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
+
+export default function TripList() {
+  const { data: trips, isLoading } = useSWR(`/api/trips`, fetcher);
+  console.log(trips);
```

Comment:
Please remember to remove the `console.log()` from before you merge this feature branch into your main branch. 🧠